community/boilerplates/serverless-triggers/aws-lambda/go/echo/README.md

```diff
@@ -0,0 +1,27 @@
+# Go Echo Example for AWS Lambda
+
+### Setup tables
+1. Create table:
+
+```
+notes:
+  id: Integer (auto-increment)
+  note: Text
+  
+  Primary key: id
+```
+
+### Setup AWS Lambda
+Create a Lambda function in AWS. This will be our webhook.
+
+1. Create a function.
+2. Select "start from scratch".
+3. Select go 1.x as the runtime.
+4. Add API gateway as a trigger.
+5. Define the API endpoint on the API gateway.
+6. Add the code in `echo.go`.
```

Comment:
There is only an option to upload zip file. Can you update the instructions accordingly?

go get the dependencies, go build, and then zipping it to make it ready for uploading. 

It'll be awesome if a small bash script can be added.